src/pyOpenMS/pxds/MetaboTargetedTargetDecoy.pxd

```diff
@@ -6,14 +6,30 @@ cdef extern from "<OpenMS/ANALYSIS/TARGETED/MetaboTargetedTargetDecoy.h>" namesp
 
   cdef cppclass MetaboTargetedTargetDecoy "OpenMS::MetaboTargetedTargetDecoy":
 
-      MetaboTargetedTargetDecoy() nogil except +
-      MetaboTargetedTargetDecoy(MetaboTargetedTargetDecoy &) nogil except + # compiler
+      MetaboTargetedTargetDecoy() nogil except + # wrap-doc:Resolve overlapping fragments and missing decoys for experimental specific decoy generation in targeted/pseudo targeted metabolomics
+      MetaboTargetedTargetDecoy(MetaboTargetedTargetDecoy) nogil except + # wrap-ignore
```

Comment:
Same here - should be:
`MetaboTargetedTargetDecoy(MetaboTargetedTargetDecoy &) nogil except + # compiler`